products/fairphone.md

```diff
@@ -0,0 +1,42 @@
+---
+permalink: /fairphone
+title: Fairphone
+category: device
+releasePolicyLink: https://forum.fairphone.com/t/fairphone-product-release-cycle/52652
+discontinuedColumn: true
+activeSupportColumn: false
+eolColumn: Supported
```

Comment:
The default value is probably better (Security Support), since that what most users are looking for. "Supported" can be confusing.

```suggestion
```